Library/Formula/couchdb.rb

```diff
@@ -98,6 +97,28 @@ def plist; <<-EOS.undent
   end
 
   def caveats; <<-EOS.undent
+    To start manually as the current user:
+        #{prefix}/bin/couchdb
+
+    If you want to automatically load `couchdb` on login:
+      First install:
+        mkdir -p ~/Library/LaunchAgents
+        cp #{prefix}/opt/couchdb/homebrew.mxcl.couchdb.plist ~/Library/LaunchAgents/
+        launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.couchdb.plist
+
+      Upgrade:
+        launchctl unload -w ~/Library/LaunchAgents/homebrew.mxcl.couchdb.plist
+        cp #{prefix}/opt/couchdb/homebrew.mxcl.couchdb.plist ~/Library/LaunchAgents/
+        launchctl load -w ~/Library/LaunchAgents/homebrew.mxcl.couchdb.plist
```

Comment:
please explain where i can find the  autogenerated caveats that you just quoted?